app/menu.ts

```diff
@@ -5,9 +5,34 @@ interface DarwinMenuItemConstructorOptions extends MenuItemConstructorOptions {
   submenu?: DarwinMenuItemConstructorOptions[] | Menu;
 }
 
+// eslint-disable-next-line import/no-default-export
 export default class MenuBuilder {
   mainWindow: BrowserWindow;
 
+  subMenuHelp: MenuItemConstructorOptions = {
+    label: 'Help',
+    submenu: [
+      {
+        label: 'Learn More',
+        click() {
+          void shell.openExternal('https://blockstack.org');
+        },
+      },
+      {
+        label: 'Documentation',
+        click() {
+          void shell.openExternal('https://docs.blockstack.org/');
```

Comment:
Maybe this could/should go to `wallet.blockstack.org` - or wherever the Wallet docs end up. I know we're still sorting that out.